canonical-web-and-design/jaas-dashboard
# Add real data to Model detail info panel

## Done

Add real data to Model detail info panel

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Go to a models detail page
- Verify the info in the models detail info panel is correct
